### ISSUE TITLE
Pitch setting missing in Simple Inference Tab ( Illaria-RVC Launcher )

### ISSUE THREAD

**Issue by Sewlell** [TheStingerX/Ilaria-RVC-Mainline#50]
Simply the title said it all. Also first.

Windows 11 + Nvidia setup

![Screenshot (433)](https://github.com/TheStingerX/Ilaria-RVC-Mainline/assets/93970226/94d597bf-21a1-43dd-9c43-751eb154e5f9)

![Screenshot (434)](https://github.com/TheStingerX/Ilaria-RVC-Mainline/assets/93970226/0907d943-26f4-493d-a069-d7df9f1043cb)

**Comment by TheStingerX** [comment]
the pitch is up there

![image](https://github.com/TheStingerX/Ilaria-RVC-Mainline/assets/108286953/cceb2f61-7734-46bc-81d8-8621f835f730)

**Comment by Sewlell** [comment]
Great, I am blind. Still testing surrounding the Launcher though.

**Comment by TheStingerX** [comment]
no worries! i had a good laugh lol

if you need anything, open an issue here or feel free to contact me on discord!
